test/unit/test-ampdoc.js

```diff
@@ -296,7 +294,16 @@ describes.sandboxed('AmpDocService', {}, () => {
 
       const content2 = document.createElement('span');
       const host2 = document.createElement('div');
-      const shadowRoot2 = host2.createShadowRoot();
+
+      let shadowRoot2;
+      if (isShadowDomSupported()) {
```

Comment:
It looks like you use shadowRoot2 below regardless of this condition. What's its use?

Reply:
That's a good observation. This was a quick fix to get master green, but could be removed